gorillaz.go

```diff
@@ -328,3 +372,26 @@ func (g *Gaz) GrpcDial(target string, opts ...grpc.DialOption) (*grpc.ClientConn
 	}
 	return grpc.Dial("gorillaz:///"+target, options...)
 }
+
+func (g *Gaz) Shutdown() {
+	Log.Info("Stopping gRPC server")
+	g.GrpcServer.Stop()
+	Log.Info("Closing gRPC listener")
+	err := g.grpcListener.Close()
+	if err != nil {
+		Log.Warn("Error while closing gRPC listener", zap.Error(err))
+	}
+	Log.Info("Shutting down http server")
+	err = g.httpSrv.Shutdown(context.Background())
+	if err != nil {
+		Log.Warn("Error while shutting down http server", zap.Error(err))
+	}
+	if g.httpListener != nil {
```

Comment:
so, the server.Shutdown() does not close the listener?

Reply:
It does.
But if it dies with 2 bullets in the back, then it looks like a suicide.